src/app/newSettings/Sections/Workspace/Members/containers/MemberDetailsContainer.tsx

```diff
@@ -196,6 +219,29 @@ const MemberDetailsContainer = ({ member, getWorkspacesMembers, isOwner, deselec
             )}
           </div>
         )}
+
+        {!isOwner && !member.isOwner && (
```

Comment:
is necessary check if the member that it is gonna leave the workspace is the owner also `member.isOwner`?
If the double check is to be kept, set the condition to a constant